docs/nextflow/bundle-scripts.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Bundling pipeline scripts
+description: Learn how Wave bundles scripts from Nextflow pipelines into container images
+date created: 2025-11-19
+date edited: 2025-11-19
+tags: [nextflow, wave, scripts, guides]
+---
+
+On cloud platforms without shared file systems, containerized tasks cannot access scripts from a shared directory. Wave solves this by packaging certain scripts directly into the container image and making them available in the task execution environment.
```

Comment:
Would be good to clarify what you mean by "shared file systems". I assume you mean something like NFS, but this could conceivably fit object storage like S3 too when viewed in a certain way.